docs/appendix.rst

```diff
@@ -143,6 +143,22 @@ This documentation uses a few special terms to refer to Python types:
       backslash-escaping any double quotes in INSTANCENAME, and putting it into
       double quotes.
 
+   default connection
+      A connection definition in the :ref:`--server general option` that is used
```

Comment:
I don't understand "in the :ref:`--server general option`" in this context.
Should this be changed to: "in the :ref:`connections file`" ?

Reply:
DONE